src/manager.rs

```diff
@@ -37,8 +37,8 @@ use crate::error::StoreError;
 
 use crate::Rkv;
 
-/// A process is only permitted to have one open handle to each Rkv environment.
-/// This manager exists to enforce that constraint: don't open environments directly.
+// A process is only permitted to have one open handle to each Rkv environment.
+// This manager exists to enforce that constraint: don't open environments directly.
```

Comment:
I think the intent here was to show this comment in the docs for Manager, although it doesn't appear anywhere in the docs when it's placed at this location. Perhaps, instead of changing it from a doc comment, we should move it to the `struct Manager` declaration.